### ISSUE TITLE
Flows stopped due to missing node types. Check logs for details.

### ISSUE THREAD

**Issue by TibGva** [DiegoViniciusCosta/node-red-contrib-s3#1]
Hi 

Thanks a lot for your node I thing it suits exactly my need as I want to store a map picture in my S3 bucket, but I need to specify the content type to png so that I can display the picture...
![issue s3 node](https://user-images.githubusercontent.com/14328511/34501776-bb21eb3e-f010-11e7-8f47-02f49523ce3e.png)

When trying to use your Amazon S3 Put node, I get the following error:

Waiting for missing types to be registered:  - aws-config

However I think I configured everything correctly 

Could this be linked to an issue with your node?

Regards
Thibaut

**Comment by DiegoViniciusCosta** [comment]
@ThibautRuy 
Hello,
I just published a update that I hope to fix this problem.
Try to remove your node, update your package.json with the last version and finally try again.
Thanks a lot for your feedback!

**Comment by TibGva** [comment]
Thanks for replying and correcting so quickly. now I get another problem:

3 Jan 21:33:15 - [warn] [amazon s3 put:Put S3] SignatureDoesNotMatch: The reques
t signature we calculated does not match the signature you provided. Check your
key and signing method.
3 Jan 21:33:15 - [error] [amazon s3 put:Put S3] AWS S3 error: SignatureDoesNotMa
tch: The request signature we calculated does not match the signature you provid
ed. Check your key and signing method.
